cmd/tapcli/addrs.go

```diff
@@ -82,7 +83,7 @@ func newAddr(ctx *cli.Context) error {
 	client, cleanUp := getClient(ctx)
 	defer cleanUp()
 
-	assetVersion, err := taprpc.MarshalAssetVersion(
+	assetVersion, err := rpcutils.MarshalAssetVersion(
```

Comment:
Maybe we can call this package `rpcmarshal` instead, so it's more clear only marshaling functions (but from any sub RPC package) should go here.